### ISSUE TITLE
Both text widgets throw TypeErrors when using language selector

### ISSUE THREAD

**Issue by jacobtylerwalls** [archesproject/arches#10122]
In the (text or Rich Text widgets), select a language from the picker.

Result:
```js
rich-text.js:81 Uncaught TypeError: Cannot set properties of undefined (setting 'value')
    at a.gc.eval [as kc] (rich-text.js:81:48)
    at Function.notifySubscribers (knockout-latest.js:42:453)
    at Function.wa (knockout-latest.js:47:255)
    at viewModel.c [as currentText] (knockout-latest.js:46:494)
    at a.gc.eval [as kc] (rich-text.js:106:12)
    at Function.notifySubscribers (knockout-latest.js:42:453)
    at Function.wa (knockout-latest.js:47:255)
    at c (knockout-latest.js:46:494)
    at Object.$a (knockout-latest.js:71:237)
    at HTMLSelectElement.k (knockout-latest.js:118:478)
```

Repeat. Second error is slightly different:

```js
rich-text.js:93 Uncaught TypeError: Cannot set properties of undefined (setting 'direction')
    at a.gc.eval [as kc] (rich-text.js:93:52)
    at Function.notifySubscribers (knockout-latest.js:42:453)
    at Function.wa (knockout-latest.js:47:255)
    at viewModel.c [as currentDirection] (knockout-latest.js:46:494)
    at a.gc.eval [as kc] (rich-text.js:107:12)
    at Function.notifySubscribers (knockout-latest.js:42:453)
    at Function.wa (knockout-latest.js:47:255)
    at c (knockout-latest.js:46:494)
    at Object.$a (knockout-latest.js:71:237)
    at HTMLSelectElement.k (knockout-latest.js:118:478)
```

Version: 7.5.dev @ 76670b00df05cea8525423548592497e7406f68a

**Comment by jacobtylerwalls** [comment]
Reproduce in regular text widget also. The consequence is that the language value changes appear as selected but aren't actually saved.

**Comment by jacobtylerwalls** [comment]
Fixed in #10140
